db/Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM postgres:latest AS sdc
+
+ENV POSTGRES_PASSWORD=Gohan123
```

Comment:
I noticed that you have passwords for this docker file with post gres and also your license key for New Relic on here.  You might want to remove these.